appserver/admingui/common/src/main/resources/js/adminjsf.js

```diff
@@ -1862,6 +1862,27 @@ function reloadHeaderFrame() {
     admingui.ajax.postAjaxRequest(mastheadForm, { render: 'af' }, 'af', false);
 }
 
```

Comment:
This function is used as when the colour is found in javascript it returns it in the format "rgb(50, 85, 200)", which needs to be converted before it can be compared to the expected value